### PR TITLE
Add ignoringFeedbackSemantics to Draggable/LongPressDraggable

### LINKED ISSUE #18640: Draggables should have a way to configure ignoring semantics of feedback widget
Dragging an item that has a11y focus unfocuses the item if it moves to feedback.  To made android behavior it should stay focused.

cc @DaveShuckerow

### PR DESCRIPTION
**Pull request by jonahwilliams**
fixes https://github.com/flutter/flutter/issues/18640

part of the work required to make a widget moved into a feedback of draggable not immediately lose a11y focus.

cc @DaveShuckerow

### CODE REVIEW

File: packages/flutter/lib/src/widgets/drag_target.dart [hunk 1]
```diff
@@ -164,6 +166,11 @@ class Draggable<T> extends StatefulWidget {
   /// Where this widget should be anchored during a drag.
   final DragAnchor dragAnchor;
 
+  /// Whether to ignore the semantics of the [feedback] widget.
+  ///
+  /// Defaults to true.
+  final bool ignoringFeedbackSemantics;
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/drag_target.dart]: It's not clear to me what this actually does or when you want to set it to false. Might need more documentation?

**Review by goderbauer — COMMENTED** [review]
Can you add tests?

File: packages/flutter/lib/src/widgets/drag_target.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter/lib/src/widgets/drag_target.dart]: Updated comment

**Comment by jonahwilliams** [comment]
Added test.  This fixes part of the re-orderable list a11y (provided you use global keys as well)

**Comment by jonahwilliams** [comment]
friendly ping @goderbauer

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**